### ISSUE TITLE
Address mismatch for mstatush

### ISSUE THREAD

**Issue by Giri2801** [riscv-software-src/riscv-config#185]
The address for mstatush is listed as 768.
In this line of [schema_isa](https://github.com/riscv-software-src/riscv-config/blob/dev/riscv_config/schemas/schema_isa.yaml#L1473)
From my understanding, it should be 784, as per RISCV priveleged spec which says address is 0x310.

**Comment by neelgala** [comment]
yes you are right. Would you mind raising a PR to the dev branch of this repo to fix the same?
